bgpd/bgp_route.h

```diff
@@ -486,12 +486,12 @@ static inline void prep_for_rmap_apply(struct bgp_path_info *dst_pi,
 
 /* called before bgp_process() */
 DECLARE_HOOK(bgp_process,
-		(struct bgp *bgp, afi_t afi, safi_t safi,
-			struct bgp_node *bn, struct peer *peer, bool withdraw),
-		(bgp, afi, safi, bn, peer, withdraw))
+	     (struct bgp * bgp, afi_t afi, safi_t safi, struct bgp_dest *bn,
```

Comment:
here the same question about `struct bgp * bgp`

Reply:
fixed